Dockerfile

```diff
@@ -0,0 +1,50 @@
+# Stage 1
+FROM mcr.microsoft.com/dotnet/sdk:6.0 as build
+WORKDIR /build
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+COPY ./Dfe.Academies.Academisation.Core/ ./Dfe.Academies.Academisation.Core/
+COPY ./Dfe.Academies.Academisation.Core.Test/ ./Dfe.Academies.Academisation.Core.Test/
+COPY ./Dfe.Academies.Academisation.Data/ ./Dfe.Academies.Academisation.Data/
+COPY ./Dfe.Academies.Academisation.Data.UnitTest/ ./Dfe.Academies.Academisation.Data.UnitTest/
+COPY ./Dfe.Academies.Academisation.Domain/ ./Dfe.Academies.Academisation.Domain/
+COPY ./Dfe.Academies.Academisation.Domain.Core/ ./Dfe.Academies.Academisation.Domain.Core/
+COPY ./Dfe.Academies.Academisation.Domain.UnitTest/ ./Dfe.Academies.Academisation.Domain.UnitTest/
+COPY ./Dfe.Academies.Academisation.IData/ ./Dfe.Academies.Academisation.IData/
+COPY ./Dfe.Academies.Academisation.IDomain/ ./Dfe.Academies.Academisation.IDomain/
+COPY ./Dfe.Academies.Academisation.IService/ ./Dfe.Academies.Academisation.IService/
+COPY ./Dfe.Academies.Academisation.IntegrationTest/ ./Dfe.Academies.Academisation.IntegrationTest/
+COPY ./Dfe.Academies.Academisation.Service/ ./Dfe.Academies.Academisation.Service/
+COPY ./Dfe.Academies.Academisation.Service.UnitTest/ ./Dfe.Academies.Academisation.Service.UnitTest/
+COPY ./Dfe.Academies.Academisation.SubcutaneousTest/ ./Dfe.Academies.Academisation.SubcutaneousTest/
+COPY ./Dfe.Academies.Academisation.WebApi/ ./Dfe.Academies.Academisation.WebApi/
+COPY ./Dfe.Academies.Academisation.WebApi.UnitTest/ ./Dfe.Academies.Academisation.WebApi.UnitTest/
+COPY ./Dfe.Academies.Academisation.sln/ ./Dfe.Academies.Academisation.sln
+
+RUN dotnet restore
+RUN dotnet build
+
+RUN dotnet new tool-manifest
+RUN dotnet tool install dotnet-ef
+
+RUN dotnet ef migrations script --output /app/SQL/DbMigrationScript.sql --idempotent -p /build/Dfe.Academies.Academisation.Data -s /build/Dfe.Academies.Academisation.WebApi -c AcademisationContext
+
+RUN dotnet publish Dfe.Academies.Academisation.WebApi -c Release -o /app
+
+COPY ./docker-entrypoint.sh /app/docker-entrypoint.sh
+
+FROM mcr.microsoft.com/dotnet/sdk:6.0 AS final
```

Comment:
would it be better not to use sdk as final container image. I've used this in past mcr.microsoft.com/dotnet/aspnet:6.0-focal and just sdk for build/test

Reply:
Good spot.

Reply:
Ah yep 👍 
I'd changed it to `sdk` because I was bug squishing, and needed the `dotnet ef` things

Using `aspnet` does mean that we have to add a step to install `curl`, but that's fine Lol